app/scrapers/__init__.py

```diff
@@ -42,6 +42,10 @@ def feed_gen(query, engine, count=10, qtype=''):
     engine = old_names.get(engine, engine)
     if engine in ('quora', 'youtube'):
         urls = scrapers[engine].search_without_count(query)
+    elif (engine in ['parsijoo']) and (qtype == 'isch'):
```

Comment:
elif engine in ['parsijoo'] and qtype == 'isch':   # Lose the parens... this is not JavaScript.